lib/active_model/serializer/adapter.rb

```diff
@@ -4,6 +4,7 @@ class Adapter
       extend ActiveSupport::Autoload
       require 'active_model/serializer/adapter/json'
       require 'active_model/serializer/adapter/json_api'
+      require 'active_model/serializer/adapter/json_sideload'
```

Comment:
@joaomdmoura  this is killing my registerable adapters pr :( https://github.com/rails-api/active_model_serializers/pull/1017